extensions/vercte/dictionaries.js

```diff
@@ -0,0 +1,142 @@
+window.dictionaries = new Map();
+class DictionaryExtension {
+  getInfo() {
+    return {
+      id: 'verctedictionaries',
+      name: 'Dictionaries',
```

Comment:
Also, since your image has a well defined color scheme, you should set the block color accordingly